CONTRIBUTING.md

```diff
@@ -0,0 +1,143 @@
+
+##### Popular Features / Feature Branches
+
+If you are implementing functionality from a known community feature branch, it is **strongly** reccomended that you open a discussion thread _before_ starting. There are some situations where maintainers would ask you to use the existing feature branch as a base, and others where maintainers would want a feature to be written from scratch.
+
+This changes on a case by case basis.
+
+#### 6. Push your changes
+When you push your first commit, you'll need to push the new branch to the remote repo.
+```bash
+git push --set-upstream origin newFeature
+```
+
+#### 7. Open Pull Request
+Once your work is complete and pushed to the branch on Github, you can open a [pull request from your branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork), targeting the branch you've chosen from `pokeemerald-expansion`. Please fill out the pull request description as completely as possible. 
+
+### What happens after I submit a pull request?
+
+A maintainer will then assign themselves as a reviewer of your pull request, and may provide feedback in the form of a PR review. 
+
+Contributors are responsible for responding to and updating their branch by addressing the feedback in the review. Contributors are also responsible for making sure the branch passes the checklist at all times.
+
+Once a maintainer has begun reviewing your PR, **please** do not force-push new changes - normal pushes are fine. Do not worry about git history - we squash most incoming changes.
+
+Maintainers will measure the submitted pull request against a [merge checklist](docs/team_procedures/merge_checklist.md).
+
+Once all items on the merge checklist are true, the branch will be merged in.
+
+
+## Maintainers
+
+This list was last updated 2025 Feb 27.
+
+| Name | Discord | Currently Active | Areas of Expertise |
+| --- | --- | --- | --- |
+| [Alex](https://github.com/AlexOn1ine) | rainonline | ✅ | Battle Engine, Battle AI
+| [Egg](https://github.com/DizzyEggg) | egg9255 | ✅ | Battle Engine, Battle AI
+| [ghoulslash](https://github.com/ghoulslash) | ghoulslash | ✅ | Dexnav, Overworld, Battle Engine
+| [Jasper](https://github.com/Bassoonian) | bassoonian | ✅ | Berries, Day / Night System, Followers, Feature Branches
+| [MGriffin](https://github.com/mrgriffin) | mgriffin | ✅ | Tests, Trainer Control
+| [psf](https://github.com/pkmnsnfrn) | pkmnsnfrn | ✅ | Rematches, Difficulty, Trainer Slides, Fake RTC, Fishing Minigames, Imperial / Metric, OW Item Balls, Sky Battles
+| [Hedara](https://github.com/hedara90) | hedara | ✅ | Compression, Sprites
+| [Pawkkie](https://github.com/Pawkkie) | pawkkie | ✅ | Battle AI
+| [SBird](https://github.com/SBird1337) | karathan | ✅ | Dynamic Multichoice, Damage Calculation, Animations, Trainer Control, Tests
+| [AsparagusEduardo](https://github.com/AsparagusEduardo) | asparaguseduardo | Inactive | 
```

Comment:
Can you remove me from this list?
I appreciate you guys thinking of me, but I thought I made it clear that I left the team outright.

Reply:
> Can you remove me from this list? I appreciate you guys thinking of me, but I thought I made it clear that I left the team outright.

nvm, opened #6521 myself.